alacritty/src/renderer/mod.rs

```diff
@@ -807,7 +808,11 @@ impl<'a> RenderApi<'a> {
     }
 
     #[inline]
-    fn add_render_item(&mut self, cell: &RenderableCell, glyph: &Glyph) {
+    fn add_render_item(&mut self, cell: &mut RenderableCell, glyph: &Glyph) {
+        cell.bg_alpha *= 255.;
```

Comment:
This can probably stay where it was? Doesn't matter by which value we multiply first, so we can keep it in the 1.0 to 0.0 range for now.

Reply:
I think there was a reason to do this, but I don't recall. I'll try whether I can figure out again why I changed that here.

Reply:
Not sure why that got changed. Seems to be fine to leave that scaling in the batch so moved it back there.